native/cocos/editor-support/spine/Extension.cpp

```diff
@@ -108,6 +112,9 @@ void DefaultSpineExtension::_free(void *mem, const char *file, int line) {
 }
 
 char *DefaultSpineExtension::_readFile(const String &path, int *length) {
+#ifdef __EMSCRIPTEN__
+    return nullptr;
+#else
```

Comment:
The code is included in `#ifndef __EMSCRIPTEN__`.

Reply:
Fixed.